inputFiles/lagrangianContactMechanics/ContactMechanics_TFrac_smoke.xml

```diff
@@ -49,7 +49,7 @@
       zCoords="{ 0, 1 }"
       nx="{ 5, 52, 5 }"
       ny="{ 5, 52, 5 }"
-      nz="{ 2 }"
+      nz="{ 3 }"
```

Comment:
this is needed to stabilize the case - otherwise it somehow runs fine locally but fails in CI